nixos/lib/test-driver/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, python3Packages
+, enableOCR ? false
+, qemu_pkg ? qemu_test
+, coreutils
+, imagemagick_light
+, libtiff
+, netpbm
+, qemu_test
+, socat
+, tesseract4
+, vde2
+}:
+
+let
+  ocrProg = tesseract4.override { enableLanguages = [ "eng" ]; };
+  imagemagick_tiff = imagemagick_light.override { inherit libtiff; };
```

Comment:
I would move these overrides to the `callPackage` site.

Reply:
moved the overrides to the callPackage site